editors/code/README.md

```diff
@@ -2,18 +2,36 @@
 
 Provides support for rust-analyzer: novel LSP server for the Rust programming language.
 
+**Note** the extension may cause conflicts with the official Rust extension. It is recommended to disable the Rust extension when using the rust-analyzer extension.
 
-Features:
+**Note** the project is in alpha status: it is already useful in practice, but can't be considered stable.
```

Comment:
I think we should finally promote it to beta :-).

Reply:
Agree, with inner items support, it does feel like beta